src/Control/Comonad/Cofree.hs

```diff
@@ -7,8 +7,13 @@
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE Safe #-}
+#else
+-- Manual Typeable instances
+{-# LANGUAGE Trustworthy #-}
 #endif
 #include "free-common.h"
+
```

Comment:
Was this whitespace-only change intended?

Reply:
Yes. I went through the modules, and now all of them have an empty line between language pragmas and the module header comment.